### PR TITLE
Automatic email

### PR DESCRIPTION
**Pull request by Ukonhattu**

### CODE REVIEW

File: backend/resolvers/Query/EmailTemplate.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      const { id } = args
+      const prisma: Prisma = ctx.prisma
+      return prisma.emailTemplate({
+        id: id,
+      })
+    },
+  })
+}
+
+const EmaiLTemplates = (t: PrismaObjectDefinitionBlock<"Query">) => {
```
Comment (nygrenh) [inline backend/resolvers/Query/EmailTemplate.ts]: Weird casing

File: backend/resolvers/Mutation/EmailTemplate.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    args: {
+      name: stringArg(),
+      html_body: stringArg(),
+      txt_body: stringArg(),
+      title: stringArg(),
+    },
+    resolve: (_, args, ctx) => {
+      checkAccess(ctx)
+      const { name, html_body, txt_body, title } = args
+      console.log(args)
```
Comment (nygrenh) [inline backend/resolvers/Mutation/EmailTemplate.ts]: Is this logging required?

File: backend/bin/kafkaConsumer/userCourseProgressConsumer/generateUserCourseProgress.ts [hunk 3]
```diff
@@ -67,6 +76,8 @@ export const generateUserCourseProgress = async (
         completion_language:
           userCourseSettings != null ? userCourseSettings.language : "unknown",
       })
+      const template = await prisma.course({ id: course.id }).completion_email()
+      if (template) sendMail(user, template)
```
Comment (nygrenh) [inline backend/bin/kafkaConsumer/userCourseProgressConsumer/generateUserCourseProgress.ts]: Can we split this to multiple lines?

**Review by nygrenh — CHANGES_REQUESTED** [review]

File: backend/resolvers/Query/EmailTemplate.ts (hunk 1 above)
Comment (Ukonhattu) [reply backend/resolvers/Query/EmailTemplate.ts]: I call it PöhinÄCase

**Review by nygrenh — APPROVED** [review]

**Pull request merged**